modules/apps/jaggery/emm/src/emm/units/role-create/role-create.js

```diff
@@ -6,11 +6,14 @@
  */
 function onRequest(context) {
     var userModule = require("/modules/user.js")["userModule"];
+    var mdmProps = require('/config/mdm-props.js').config();
     var response = userModule.getRoles();
     if (response["status"] == "success") {
         context["roles"] = response["content"];
     }
     var userStores = userModule.getSecondaryUserStores();
     context["userStores"] = userStores;
+    context["rolenameJSRegEx"] = mdmProps.roleValidationConfig.rolenameJSRegEx;
+    context["rolenameRegExViolationErrorMsg"] = mdmProps.roleValidationConfig.rolenameRegExViolationErrorMsg;
```

Comment:
I have used the following naming convention {{elementID}}+JSRegEx and {{elementID}}+eRegExViolationErrorMsg.
rolename is used since its the id of the related HTML element.
can be change to roleNameJSRegEx by using java script variable naming convention.
WDYT?